sdk/communication/AzureCommunicationChat/Source/Registrar/RegistrarSettings.swift

```diff
@@ -45,17 +45,17 @@ internal enum RegistrarSettings {
     static let context = ""
 }
 
-internal enum RegistrarHeader {
+internal enum RegistrarHeader: String {
     /// Content-type header.
-    static let contentType = "Content-Type"
+    case contentType = "Content-Type"
     /// Skype token for authentication.
-    static let skypeTokenHeader = "X-Skypetoken"
+    case skypeTokenHeader = "X-Skypetoken"
 }
 
-internal enum RegistrarMimeType {
-    static let json = "application/json"
+internal enum RegistrarMimeType: String {
+    case json = "application/json"
 }
 
-internal enum RegistrarStatusCode {
-    static let success = 202
+internal enum RegistrarStatusCode: Int {
+    case success = 202
```

Comment:
Are you actually referencing these anywhere?

Reply:
Yep. RegistrarMimeType is referenced in "RegistrarClient - setRequestHeader()". RegistrarStatusCode is referenced in "RegistrarClientTest" and "ChatClientDVRTest".

Reply:
That's fine, but you have other instances were 202 is hard-coded instead of referencing this enum. Also, HTTP status codes are generic across all of HTTP. They can't be tailored to Registrar specifically, so just the name of this enum seems odd.

Reply:
Please let me know which option you recommend: (1)rename the "RegistrarStatusCode" into "HTTPStatusCode"; (2)hardcode all "RegistrarStatusCode" into "202"

Reply:
Since this is all internal, either of these options are fine to me:
1. Hard-code 202 and remove `RegistrarStatusCode`
2. Rename to `HTTPStatusCode` and replace hard-coded 202s with this enum value.

Reply:
Thanks. I chose option 1 and fixed the codes. Will push the change later.